components/event_dispatcher.rst

```diff
@@ -325,7 +325,7 @@ Using Event Subscribers
 
 The most common way to listen to an event is to register an *event listener*
 with the dispatcher. This listener can listen to one or more events and
-is notified each time those events are dispatched.
+it will be notified each time those events are dispatched.
```

Comment:
I'm not sure about that change, the usage of present looks quite right to me here, but I'm French so it may feel more natural to me to use present for absolute truth.